packages/shopify-codemod/README.md

```diff
@@ -5,9 +5,10 @@ This repository contains a collection of Codemods written with [JSCodeshift](htt
 ## Usage
 
 1. `npm install -g jscodeshift`
-1. `git clone https://github.com/shopify/javascript` or [download the zip file](https://github.com/shopify/javascript/archive/master.zip)
-1. `jscodeshift -t <codemod-script> <file>`
-1. Use the `-d` option for a dry-run and use `-p` to print the output for comparison.
+2. `git clone https://github.com/shopify/javascript` or [download the zip file](https://github.com/shopify/javascript/archive/master.zip)
+3. `npm install && npm run setup`
```

Comment:
Is it better to have `jscodeshift` as a local dependency? The reason is similar to why Babel doesn't recommend being installed globally:

1. Different projects on the same machine can depend on different versions of Babel allowing you to update one at a time.
2. It means you do not have an implicit dependency on the environment you are working in. Making your project far more portable and easier to setup.